es/ogn_decode_parser.py

```diff
@@ -0,0 +1,96 @@
+import re
+import sys
+import datetime
+from elasticsearch import Elasticsearch
+import json
+import argparse
+
+# Requires elasticsearch python package
+# > pip install elasticsearch
+# Probably also requires some other stuff like perl
+
+# Global configuration:
+# Elastic type name, mostly don't care so naming it loc
+elastic_type = "loc"
+
+
+def main():
+	global es, args
+
+	parser = argparse.ArgumentParser(description="Parse OGN decode stream and upload to ES")
+	parser.add_argument("--dry-run", action="store_true", help="Do not upload")
+	parser.add_argument("--input-file", help="Read OGN decode data from file instead of stdin")
+	parser.add_argument("--no-pass-through", help="Do not print out the decode stream during decoding", action="store_true")
+	parser.add_argument("--es-server", help="URL:port to elasticsearch", default="localhost:9200")
+	args = parser.parse_args()
+
+	
+	# Setup
+	es = Elasticsearch([args.es_server])
```

Comment:
vi får kanskje trekke ut es-parametre (server, brukernavn, passord) i en json fil for både .rb og .py-scriptene